src/isql/isql.epp

```diff
@@ -9597,33 +9597,24 @@ static const char* charset_to_string(unsigned charset)
  **************************************/
 	static Firebird::GlobalPtr<Firebird::GenericMap<Firebird::Pair<Firebird::Right<unsigned, string> > > > csMap;
 
-	charset = TTYPE_TO_CHARSET(charset);
```

Comment:
What is the relation of this isql changes with #8249 ?

Reply:
QA team and me should have a way to check field collation to make sure that #8249 really fixed and result of the `CAST()` has the right collation. The simplest way - to print collation in SQLDA along to the charset. That's why this change.

Reply:
First, ISQL showed the charset/collation id, so it was identifiable.
You want to display also the collation name, but this change makes it incorrectly. Collation ids, different from charset ids, are far from be stable between different databases. I suggest to revert this part of the patch.